sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/Utils.java

```diff
@@ -51,17 +20,9 @@
 final class Utils {
```

Comment:
Can this all just be moved into `UtilsImpl` to further reduce accidentally exposing implementation utilities in the public API

Reply:
Done.